convert/llama.go

```diff
@@ -161,16 +161,9 @@ func (m *LlamaModel) WriteGGUF() (string, error) {
 
 	f, err := os.CreateTemp("", "ollama-gguf")
```

Comment:
Looks like you might have missed removing the `CreateTemp` here, it is removed in the other places

Reply:
Good catch. This isn't using the WriteSeeker which is a bug